features/unit/feetest.feature

```diff
@@ -0,0 +1,42 @@
+      | delete        | 100             | BH55E5RMBD4GYWXGX5W5PJ5JAHPGM5OXKDQH5DC4O2MGI7NW4H6VOE4CP4     |                    |                 | 0            | 0           | 0           | 0          | str:test       | 5555,6666           | 7777,8888           | AAVDEAJ3NIYOG7XCRBKCJ3T5PUCVL2XASOP3NGX4NPPZ3UX6477PBG6E4Q,AADQIC4PMKRTFMHAAXYAFSGAUULDI2ABBIYVQJ6GZ5JHY6DJPHTU2SPHYM  | 1234          | 9000         | 9010  | Mf0h6zjkEIEZPtNM3zsrg+iHQFS0fZxhgr7w35I464M= | gqNzaWfEQNRuI7jjkY7nVUFd+weVjF9vL80VLDDhOZTZ7Iu6gPpvMzgiaKhRWPw2GdvdggqyqAp6R71B0iNItmTKcS6ZhAmjdHhujKRhcGFhkcQEdGVzdKRhcGFuBaRhcGFzks0eYc0iuKRhcGF0ksQgACoyATtqMON+4ohUJO59fQVV6uCTn7aa/GvfndL+5/7EIAAHBAuPYqMysOAF8ALIwKUWNGgBCjFYJ8bPUnx4aXnnpGFwZmGSzRWzzRoKpGFwaWRko2ZlZc0E0qJmds0jKKJnaMQgMf0h6zjkEIEZPtNM3zsrg+iHQFS0fZxhgr7w35I464OibHbNIzKjc25kxCAJ+9J2LAj4bFrmv23Xp6kB3mZ111Dgfoxcdphkfbbh/aR0eXBlpGFwcGw= | 0 |
+
+
+  Scenario Outline: Payment Txn Fee Test
+    Given payment transaction parameters <fee> <fv> <lv> "<gh>" "<to>" "<close>" <amt> "<gen>" "<note>"
+    And mnemonic for private key "<mn>"
+    And multisig addresses "<addresses>"
+    When I create the multisig payment transaction
+    And I sign the multisig transaction with the private key
+    Then fee field is in txn
+    Given payment transaction parameters 0 <fv> <lv> "<gh>" "<to>" "<close>" <amt> "<gen>" "<note>"
+    And mnemonic for private key "<mn>"
+    And multisig addresses "<addresses>"
+    When I create the multisig payment transaction with zero fee
+    And I sign the multisig transaction with the private key
+    Then fee field not in txn
+
+    Examples:
+    | fee | fv    | lv    | gh                                           | to                                                         | close                                                      | amt  | gen          | note         | mn                                                                                                                                                                   | addresses                                                                                                                                                                        | golden                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                           |
```

Comment:
Nit: I don't think `golden` is used here or in the previous scenario -- you could remove it.